profiles/kentik_snmp/broadforward/diameter.yml

```diff
@@ -0,0 +1,168 @@
+# This profile matches on the generic Linux SysOID
+# https://oidref.com/1.3.6.1.4.1.8072.3.2.10
+---
+extends:
+  - host-resources-mib.yml
+  - if-mib.yml
+  - ip-mib.yml
+  - system-mib.yml
+  - ucd-mib.yml
+
+# We may look at building a new entity type for these devices if needed
+provider: kentik-net-snmp
+
+# Do not want this profile to automatically get applied to other linux servers, hard coded only
+# sysobjectid:
+
+metrics:
+  - MIB: MF-DIAMETER-MIB
+    table:
+      OID: 1.3.6.1.4.1.39216.1.2.20
+      name: dmtrPeerTable
+    symbols:
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.1.2
+        name: dmtrPeerAdminStatus
+        enum:
+          - inactive: 0
+          - active: 1
+      - OID: 1.3.6.1.4.1.39216.1.50.1.2.1.3
+        name: dmtrPeerDiameterId
```

Comment:
this is a string-value, we should move it to `metric_tags`

Reply:
right